### PR TITLE
Billy/1073 bar bond

### LINKED ISSUE #1073: Bond Bar out of margin
UI Version: 0.9.3

Description:

Disable or hide bond bar if the user is revoked.
See picture below.

screen shot 2018-08-02 at 1 26 22 pm

### PR DESCRIPTION
**Pull request by okwme**
Closes #1073 

Description: added more checks since it's possible to type in numbers outside of the input range and removed the draggable handle.

### CODE REVIEW

File: app/src/renderer/components/staking/PageBond.vue [hunk 1]
```diff
@@ -125,7 +126,6 @@ tm-page.page-bond(:title="`Bond ${denom}`")
 import { between, numeric, required } from "vuelidate/lib/validators"
 import { mapGetters } from "vuex"
 import num from "scripts/num"
-import interact from "interactjs"
```
Comment (faboweb) [inline app/src/renderer/components/staking/PageBond.vue]: do we still need the interactjs dependency?

Comment (okwme) [reply app/src/renderer/components/staking/PageBond.vue]: good call! i'll remove

File: app/src/renderer/components/staking/PageBond.vue [hunk 2]
```diff
@@ -286,6 +286,7 @@ export default {
     bondBarPercent(dividend) {
       let divisor = this.totalAtoms
       let ratio = Math.round(dividend / divisor * 100)
+      if (isNaN(ratio)) ratio = 0
```
Comment (NodeGuy) [inline app/src/renderer/components/staking/PageBond.vue]: This is OK but next time it's better to write like this:

```
if (isNan(ratio)) { ratio = 0 }
```

That calls out more clearly that there's a side effect being caused.

File: app/src/renderer/components/staking/PageBond.vue [hunk 3]
```diff
@@ -294,6 +295,7 @@ export default {
       let divisor = this.totalAtoms
       let ratio = Math.round(dividend / divisor * 100) / 100
       let width = ratio * (maxWidth - offset) + offset
+      if (width > maxWidth) width = maxWidth
```
Comment (NodeGuy) [inline app/src/renderer/components/staking/PageBond.vue]: Same as above.

File: app/src/renderer/components/staking/PageBond.vue (hunk 2 above)
Comment (okwme) [reply app/src/renderer/components/staking/PageBond.vue]: 👍

File: app/src/renderer/components/staking/PageBond.vue (hunk 3 above)
Comment (okwme) [reply app/src/renderer/components/staking/PageBond.vue]: 👍

**Pull request merged**